csv/transformer.go

```diff
@@ -6,79 +6,81 @@ import (
 )
 
 type ReverseTransformer struct {
-	defaultTransformer destination.DefaultReverseTransformer
+	*destination.DefaultReverseTransformer
 }
 
-func (t *ReverseTransformer) ReverseTransformValues(table *schema.Table, values []any) (schema.CQTypes, error) {
-	return t.defaultTransformer.ReverseTransformValues(table, values)
-}
+var _ interface {
+	ReverseTransformValues(table *schema.Table, values []any) (schema.CQTypes, error)
```

Comment:
Why do we need / want this interface declaration? I get that it's essentially asserting that the interface is implement at compile-time, and it makes sense to me for the `schema.CQTypeTransformer` case, but why the custom interface here?

Reply:
Just wanted to make sure it works fine statically.
Basically, we should either have this interface declared in sdk or just drop